tests/test_tasks.py

```diff
@@ -0,0 +1,108 @@
+from rest_framework import status
+from .common import EndpointTestCase
+
+
+class TestTasksGet(EndpointTestCase):
+    """
+    Tests for '/api/students/<int:student_id>/tasks/' [GET] endpoint.
+    """
+
+    # Fixture specific URL to available student data.
+    VALID_URL = "/api/students/2/tasks/"
+    # Fixture specific URL to data not available for current user.
+    NOT_PERMITTED_URL = "/api/students/1/tasks/"
+    # Fixture specific URL to invalid student ID.
+    NOT_FOUND_URL = "/api/students/12345/tasks/"
+
+    def test_Get_Success(self):
```

Comment:
If we want to change the convention and group tests in `Test<ENDPOINT><METHOD>` classes, then remove `_Get` from each method name.

Reply:
I changed name of method

Reply:
This is not fixed.

Reply:
just have fixed this one